spec-main/api-net-spec.ts

```diff
@@ -477,160 +393,120 @@ describe('net module', () => {
         response.statusMessage = 'OK'
         response.end()
       })
+
       const urlRequest = net.request(serverUrl)
-      const complete = new Promise<number>(resolve => {
-        urlRequest.on('response', (response) => {
-          resolve(response.statusCode)
-          response.on('data', () => {})
-          response.on('end', () => {})
-        })
-      })
       urlRequest.setHeader(customHeaderName, customHeaderValue as any)
       expect(urlRequest.getHeader(customHeaderName)).to.equal(customHeaderValue)
       expect(urlRequest.getHeader(customHeaderName.toLowerCase())).to.equal(customHeaderValue)
       urlRequest.write('')
       expect(urlRequest.getHeader(customHeaderName)).to.equal(customHeaderValue)
       expect(urlRequest.getHeader(customHeaderName.toLowerCase())).to.equal(customHeaderValue)
-      urlRequest.end()
-      expect(await complete).to.equal(200)
+      const response = await getResponse(urlRequest)
+      expect(response.statusCode).to.equal(200)
+      await collectStreamBody(response)
```

Comment:
if we don't need to assert anything about the result, can we just drop it?

Reply:
Answered above